content/docs/user-guide/building/venv.md

```diff
@@ -0,0 +1,54 @@
+# Virtual Environments
+
+Given a model and a list of its dependencies and packages, an environment needs
+to be present that has these requirements readily available so as to use the
+model. To make sure that different dependencies for different models (or
+projects) don't clash,
+[virtual environments](https://realpython.com/python-virtual-environments-a-primer/)
+are used.
```

Comment:
Shouldn't this also be a section in [UG/ Building/ Requirements](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building/requirements) ?

Reply:
Link from somewhere in [Python Packages](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building/pip)?

Reply:
> Shouldn't this also be a section in [UG/ Building/ Requirements](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building/requirements) ?

The `Description` section already has a link to `UG / Requirements`

Reply:
> Link from somewhere in [Python Packages](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building/pip)?

not sure if this is related. Because the guide linked above is to create a python package FOR an mlem model, and not to install python packages REQUIRED by an mlem model.